recipe/meta.yaml

```diff
@@ -78,27 +97,42 @@ outputs:
     version: {{ sdk_version }}
     script: build-sdk.bat  # [win]
     script: build-sdk.sh  # [not win]
+    build:
+      run_exports:
+        - {{ pin_subpackage("dotnet-runtime", max_pin="x.x") }}
+    #   missing_dso_whitelist:
+    #     - "$RPATH/ld-linux-x86-64.so.2"  # [linux and aarch64]
+    #   ignore_run_exports:
+    #     - libcurl  # [unix]
+    #     - libcxx  # [osx]
+    #     - libzlib  # [osx]
+    #     - lttng-ust  # [linux]
     requirements:
-      build:
-        - {{ stdlib('c') }}
-        - {{ compiler('cxx') }}
-        - {{ stdlib('c') }}
-        - posix                  # [win]
+      # build:
+      #   - {{ compiler('cxx') }}
+      #   - {{ stdlib('c') }}
+      #   - posix  # [win]
       host:
-        - zlib                   # [unix]
-        - libcurl                # [unix]
-        - lttng-ust              # [linux64]
+        - dotnet-runtime ={{ runtime_version }}
+      #   - libcurl  # [unix]
+      #   - lttng-ust  # [linux64]
+      #   - zlib  # [unix]
       run:
-        - lttng-ust             # [linux64]
-        - libcurl               # [unix]
-        - icu                   # [unix]
-        - krb5                  # [unix]
-        - __glibc >=2.17        # [linux64]
         - dotnet-runtime ={{ runtime_version }}
     test:
       commands:
-        - dotnet --version                   # [not aarch64]
+        - dotnet --version  # [not aarch64]
         - test -f $PREFIX/lib/dotnet/dotnet  # [aarch64]
+        - export DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1  # [aarch64]
+        - mkdir self-test && cd self-test && dotnet new console && dotnet clean  # [unix]
+        - cat self-test.csproj  # [unix]
+        - DOTNET_EnableMacOSCodeSign=false dotnet build && dotnet exec bin/Debug/net{{ framework }}/self-test.dll > self-test.txt  # [unix]
```

Comment:
Thanks for making this flexible with the framework variable!

Reply:
I eventually found your post of 2021 on this exact same issue, it seems that I am walking in your footsteps